File: dev/requirements/screen/cases/setup-highlight.3.3.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 3.3 — the crucial setup picked out of the body in orange.
let setupHighlight_3_3 = RequirementCase(id: "3.3") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "setup-highlight", id: "3.3", region: .jokeBody)
    }
}
#endif

File: dev/requirements/screen/cases/setup-highlight.3.3.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 3.3 — Crucial setups render highlighted inside the body: orange and bold, against
/// the body's bone.
final setupHighlight_3_3 = RequirementCase('3.3', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.jokeBody,
    slug: 'setup-highlight',
    id: '3.3',
  );
});
